### PR TITLE
DT-1191: 🎨 Reformat and set java editorconfig too

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

**Review by JamieDraperUK — APPROVED** [review]
LGTM

File: .editorconfig [hunk 1]
```diff
@@ -1,3 +1,9 @@
 [*.{kt,kts}]
-indent_size=2
-max_line_length=120 
+indent_size = 2
+max_line_length = 120
```
Comment (petergphillips) [inline .editorconfig:3]: you can't tell, but there is a trailing space on this line temporarily.  This causes the line length to be ignored, will fix in next PR and reformat the kotlin code again to cope.

File: .editorconfig [hunk 2]
```diff
@@ -1,3 +1,9 @@
 [*.{kt,kts}]
-indent_size=2
-max_line_length=120 
+indent_size = 2
+max_line_length = 120 
+
+[*.java]
+ij_java_class_count_to_use_import_on_demand = 99
```
Comment (petergphillips) [inline .editorconfig:6]: add in default settings for imports

**Review by JamieDraperUK — APPROVED** [review]
lgtm

**Pull request merged**